src/main/kotlin/tanvd/grazi/grammar/LangToolFixes.kt

```diff
@@ -0,0 +1,24 @@
+package tanvd.grazi.grammar
+
+import org.languagetool.rules.Rule
+
+enum class LangToolFixes(private val ruleId: String, private val fix: (suggestion: String) -> String) {
+    ARTICLE_MISSING("ARTICLE_MISSING", { suggestion ->
+        when {
+            suggestion.startsWith("The") -> "The " + suggestion[4].toLowerCase() + suggestion.substring(5)
```

Comment:
What if it has more that one whitspace?

Reply:
There is no possibility to get more than one whitespace, btw added regex to replace it